app/src/main/java/com/vero/tasky/agenda/data/remindermanager/ReminderManagerImpl.kt

```diff
@@ -0,0 +1,46 @@
+package com.vero.tasky.agenda.data.remindermanager
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import com.vero.tasky.agenda.data.broadcastreceiver.ReminderReceiver
+import com.vero.tasky.agenda.data.util.LocalDateTimeConverter
+import com.vero.tasky.agenda.domain.remindermanager.ReminderData
+import com.vero.tasky.agenda.domain.remindermanager.ReminderManager
+
+class ReminderManagerImpl(
```

Comment:
This reminder isnt related to the Agenda ítem reminder, right?

In this case, i'd find a different name. Since its confusing

Something like "AlarmHandler" or similar